devices/atlantic.js

```diff
@@ -0,0 +1,157 @@
+        key: ['presence_detection'],
+        convertSet: async (entity, key, value, meta) => {
+            assert(typeof value === 'boolean');
+            await entity.write('hvacThermostat', {0x4275: {value: value ? 1 : 0, type: 0x30}}, {manufacturerCode: 0x125b});
+            return {state: {presence_detection: value}};
+        },
+    },
+    quiet_fan: {
+        key: ['quiet_fan'],
+        convertSet: async (entity, key, value, meta) => {
+            assert(typeof value === 'boolean');
+            await entity.write('hvacFanCtrl', {0x1000: {value: value ? 1 : 0, type: 0x10}}, {manufacturerCode: 0x125b});
+            return {state: {quiet_fan: value}};
+        },
+    },
+    ac_louver_position: {
+        key: ['ac_louver_position'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(thermostatPositions));
+            const index = thermostatPositions[value];
+            if (index === thermostatPositions.swing) {
+                await entity.write('hvacThermostat', {0x4274: {value: 1, type: 0x10}}, {manufacturerCode: 0x125b});
+            } else {
+                await entity.write('hvacThermostat', {0x4274: {value: 0, type: 0x10}}, {manufacturerCode: 0x125b});
+                await entity.write('hvacThermostat', {0x4273: {value: index, type: 0x30}}, {manufacturerCode: 0x125b});
+            }
+            return {state: {ac_louver_position: value}};
+        },
+    },
+    energy_mode: {
+        key: ['energy_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(energyMode));
+            const index = energyMode[value];
+            switch (index) {
+            case energyMode.eco:
+                await entity.write('hvacThermostat', {0x4270: {value: 0, type: 0x10}}, {manufacturerCode: 0x125b});
+                await entity.write('hvacThermostat', {'programingOperMode': 4});
+                break;
+            case energyMode.normal:
+                await entity.write('hvacThermostat', {'programingOperMode': 0});
+                await entity.write('hvacThermostat', {0x4270: {value: 0, type: 0x10}}, {manufacturerCode: 0x125b});
+                break;
+            case energyMode.powerful:
+                await entity.write('hvacThermostat', {'programingOperMode': 0});
+                await entity.write('hvacThermostat', {0x4270: {value: 1, type: 0x10}}, {manufacturerCode: 0x125b});
+                break;
+            }
+            return {state: {energy_mode: value}};
+        },
+    },
+    thermostat_programming_operation_mode: {
+        key: ['programming_operation_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            await tz.thermostat_programming_operation_mode.convertSet(entity, key, value, meta);
+            return {state: {programingOperMode: value}};
```

Comment:
Same as https://github.com/Koenkk/zigbee-herdsman-converters/pull/4560/files#r956350901, but change `programingOperMode` to `programming_operation_mode`